### ISSUE TITLE
Remove Round Files Settings that are non-operational

### ISSUE THREAD

**Issue by MichaelNMaggs** [hatnote/montage#133]
Most of the available tick boxes in Round Files Settings are non-operational, according to https://commons.wikimedia.org/wiki/Commons:Montage. Specifically, "Disqualify coordinators", "Disqualify organizers" and "Show link" are said not to work. The documentation does not say whether or not "Disqualify jurors" and "Disqualify maintainers" do anything.

Users shouldn't be presented with options that are known not to work.

**Comment by leilaz** [comment]
@yarl can you remove these options? They are quite confusing at the moment. (feel free to remove all for now)

**Comment by slaporte** [comment]
Agreed, we should remove these options until the feature is fully designed/supported.

**Comment by MichaelNMaggs** [comment]
Still causing confusion

**Comment by lgelauff** [comment]
I believe this has now been dealt with - but please open new reports when you come across any specifics!
